docs/demos/on-click/demo.js

```diff
@@ -0,0 +1,55 @@
+import { Checkboxland } from '../../../src/index.js';
+
+let dimensions = '44x15';
+let cbl;
+let lastPos = null;
+
+function init(existingCbl) {
+  cbl = !!existingCbl ? existingCbl : new Checkboxland({ dimensions });
+
+  const data = cbl.getData();
+  cbl.setData(data.map(e => e.map(_ => 0)));
```

Comment:
This works fine (no need to update this), but just to let you know, you could also use [`cbl.clearData()`](https://www.bryanbraun.com/checkboxland/#cleardata) for this. 👍